samcli/local/docker/manager.py

```diff
@@ -126,7 +126,7 @@ def stop(self, container: Container) -> None:
             container.stop()
         container.delete()
 
-    def pull_image(self, image_name, stream=None):
+    def pull_image(self, image_name, tag="latest", stream=None):
```

Comment:
Should we add any validation that we only pull latest in some manner?

Reply:
Tests will be added in another PR. This PR is against the feat/public-ecr feature branch. Trying out using feature branches to see if there are issues.